include/ozo/thread_safety.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
```

Comment:
It is better to move the header into the `core` folder

Reply:
Done